pop/popmw/tx_test.go

```diff
@@ -47,81 +56,101 @@ func tx(fn func(tx *pop.Connection)) error {
 	if err := db.RawQuery(mig).Exec(); err != nil {
 		return err
 	}
+
 	fn(db)
+
 	return nil
 }
 
 func app(db *pop.Connection) *buffalo.App {
 	app := buffalo.New(buffalo.Options{})
 	app.Use(Transaction(db))
-	app.GET("/success", func(c buffalo.Context) error {
+
+	// all handlers do the same database job but the return differently
+	app.GET("/success-201", func(c buffalo.Context) error {
 		w := &widget{}
 		tx := c.Value("tx").(*pop.Connection)
 		if err := tx.Create(w); err != nil {
 			return err
 		}
-		return c.Render(201, nil)
+		return c.Render(201, nil) // 201 created
```

Comment:
@sio4 we could change these numbers into (in this case) `http.StatusCreated` and avoid the comment.

https://pkg.go.dev/net/http#pkg-constants

Reply:
This is nothing major I agree. Is just that the comment made me think you wanted to indicate that's a Created status. I'm fine with it.